### PR TITLE
feat(environments): Add environment toggle on Releases page

### PR DESCRIPTION
**Pull request by lynnagara**
Also add withEnvironment higher order component

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/projectReleases/index.jsx [hunk 1]
```diff
@@ -75,7 +77,22 @@ const ProjectReleases = createReactClass({
       error: false,
     });
 
-    this.api.request(this.getProjectReleasesEndpoint(), {
+    const {orgId, projectId} = this.props.params;
+
+    const url = `/projects/${orgId}/${projectId}/releases/`;
+
+    let query = {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectReleases/index.jsx]: `const`?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectReleases/index.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/projectReleases/index.jsx]: oops

**Comment by lynnagara** [comment]
What do you mean by allowlist?

**Pull request merged**